tests/video_games_sales/test_data_processor.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for the DataProcessor class."""
+
+import numpy as np
+
+from src.video_game_sales.data_processor import DataProcessor
```

Comment:
typically, inside src folder you have another folder with project name  (for example, video_games)